conf/svcomp.json

```diff
@@ -89,11 +89,6 @@
     }
   },
   "witness": {
-    "graphml": {
-      "enabled": true,
```

Comment:
If removing this part entirely, instead of switching to `false`, then one might as well remove it from svcomp-validate.json as well.

Reply:
is there any advantage to keeping it around and switching it to `false`?

Reply:
Not really. I just had it explicit in svcomp-validate.json to make it more obvious in the diff of the two confs.